src/Bicep.Decompiler.IntegrationTests/Working/101-1vm-2nics-2subnets-1vnet/azuredeploy.bicep

```diff
@@ -1,36 +1,18 @@
-param virtualMachineSize string {
-  metadata: {
-    description: 'Virtual machine size (has to be at least the size of Standard_A3 to support 2 NICs)'
-  }
-  default: 'Standard_DS1_v2'
-}
-param adminUsername string {
-  metadata: {
-    description: 'Default Admin username'
-  }
-}
-param adminPassword string {
-  metadata: {
-    description: 'Default Admin password'
-  }
-  secure: true
-}
-param storageAccountType string {
-  allowed: [
-    'Standard_LRS'
-    'Premium_LRS'
-  ]
-  metadata: {
-    description: 'Storage Account type for the VM and VM diagnostic storage'
-  }
-  default: 'Standard_LRS'
-}
-param location string {
-  metadata: {
-    description: 'Location for all resources.'
-  }
-  default: resourceGroup().location
-}
+@description('Virtual machine size (has to be at least the size of Standard_A3 to support 2 NICs)')
+param virtualMachineSize string = 'Standard_DS1_v2'
+@description('Default Admin username')

```

Comment:
This generated output is now pretty hard to read without spacing between parameters. What do you think about including an extra newline after each param declaration?

Reply:
Yeah totally agree. I'll update the json files.

Reply:
I add a newline before each parameter declaration with decorators so that those without decorators can still stick together. Let me know if this approach works.

Reply:
Looks good to me!